spec/dummy/app/assets/config/manifest.js

```diff
@@ -2,4 +2,3 @@
 //= link_tree ../images
 //= link_directory ../javascripts .js
 //= link_directory ../stylesheets .css
-//= link terms_manifest.js
```

Comment:
This was failing on Rails 6 tests, could not figure out reasoning behind this file, seems phantom. Also, super weird that the versions of rails prior to 6.0 passed without error.